xapi_db_load/backends/clickhouse_lake.py

```diff
@@ -20,9 +20,10 @@ def __init__(self, db_host="localhost", db_port=18123, db_username="default",
             password=db_password,
             port=self.port,
             database=self.database,
-            date_time_input_format="best_effort",  # Allows RFC dates
-            old_parts_lifetime=10,  # Seconds, reduces disk usage
```

Comment:
@bmtcril I get this error when I include `'old_parts_lifetime': 10` in the client settings:

> clickhouse_connect.driver.exceptions.ProgrammingError: Setting old_parts_lifetime is unknown or readonly

So I've added it to the MergeTable CREATE statement below instead. From what you've learned about Clickhouse, is that ok?

Reply:
That should be fine here. I wonder if there's a version mismatch in one of our installs, though. If we're going to be using this elsewhere we should probably do the work to make it an officially supported repo. I'll ask around about what people think about that.

Reply:
@bmtcril Or maybe we need to pin the version of `clickhouse-connect`? I've got this, what are you running?

```
clickhouse-connect==0.5.3
```

Reply:
I've got 0.3.6 so am on the other side of this breaking change: https://github.com/ClickHouse/clickhouse-connect/blob/main/CHANGELOG.md#050-2023-01-14

I think pinning it to >0.5 and <0.6 is probably the best since they seem to do breaking changes on minor versions.